Lib/fontTools/varLib/varStore.py

```diff
@@ -37,12 +41,14 @@ def setSupports(self, supports):
 		self._cache = {}
 		self._data = None
 
-	def finish(self, optimize=True):
+	def finish(self, optimize=True, empty=True):
 		self._regionList.RegionCount = len(self._regionList.Region)
 		self._store.VarDataCount = len(self._store.VarData)
 		for data in self._store.VarData:
 			data.ItemCount = len(data.Item)
 			data.calculateNumShorts(optimize=optimize)
+		if empty is None and not self._store.VarDataCount:
+			return None
```

Comment:
it doesn't seem like you're using this empty=None parameter you just added anywhere. Where did you think it could be handy?

Reply:
I don't use it currently. I thought in some situations we can do like:
```
  table.VarStore = varStore.optimize(empty=None)
```
But am fine dropping that commit.

Reply:
i see. Well, until we find a use for it, i'd remove the extra code paths

Reply:
Okay. Let me do that.